packages/compat/src/v1-app.ts

```diff
@@ -23,18 +23,18 @@ import { isEmberAutoImportDynamic } from './detect-ember-auto-import';
 // This controls and types the interface between our new world and the classic
 // v1 app instance.
 
-type filePath = string;
-type OutputFileToInputFileMap = { [filePath: string]: filePath[] };
+type FilePath = string;
+type OutputFileToInputFileMap = { [filePath: string]: FilePath[] };
```

Comment:
Under the new `@typescript-eslint/naming-convention` rule settings, PascalCase is required for all so called `typeLike` things, including type aliases. We could fine tune this further to only include classes and interfaces as it was before, but this seems more consistent to me, and matches the default config of that rule, and needed changes only in these two files.